Essentials/src/config.yml

```diff
@@ -758,6 +758,30 @@ protect:
     # Monsters won't follow players.
     # permission essentials.protect.entitytarget.bypass disables this.
     entitytarget: false
+    # Prevent certain transformations.
+    transformation:
+      creeper-charge: false
+      villager-infection: false
+      villager-cure: false
+      villager-to-witch: false
+      pig-transformation: false
+      zombie-drowning: false
+      # Prevent brown and red mooshrooms switch on lightning strike
```

Comment:
It looks like GitHub's suggestion didn't apply properly - these lines can be removed.

Reply:
Woops, got that fixed.